plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -443,8 +448,11 @@ public boolean isSupportedPartitionKey()
             case DOUBLE:
             case INET:
             case INT:
+            case TINYINT:
+            case SMALLINT:
```

Comment:
Seems like these two types already had the necessary code implemented?

Reply:
Yes, I simply forgot updating this switch statements when adding those types.